services/startup.sh

```diff
@@ -25,6 +25,9 @@ touch "${once}"
 # Use default directory if SERVICE_DIR is not provided.
 SERVICE_DIR="${SERVICE_DIR-/media/developer/apps/usr/palm/services/org.webosbrew.hbchannel.service}"
 
+# Add service bin directory to PATH for devices without telnetd.
+export PATH="${PATH}:${SERVICE_DIR}/bin"
```

Comment:
Remember to add a relevant checksum in [service.ts](https://github.com/webosbrew/webos-homebrew-channel/blob/81d3d576fc63daa73c61561182197f478c91b92b/services/service.ts#L605-L612)